lib/provider.coffee

```diff
@@ -131,7 +131,7 @@ module.exports =
     scopes = scopeDescriptor.getScopesArray()
     if hasScope(scopes, 'meta.selector.css') and not hasScope(scopes, 'source.sass')
       true
-    else if hasScope(scopes, 'source.css.scss') or hasScope(scopes, 'source.css.less') or hasScope(scopes, 'source.sass')
+    else if hasScope(scopes, 'source.css.scss') or hasScope(scopes, 'source.css.stylus') or hasScope(scopes, 'source.css.less') or hasScope(scopes, 'source.sass')
```

Comment:
This isn't in the same order as the one above - can you please make them consistent?  Thanks.